lib/vagrant-libvirt/action/create_networks.rb

```diff
@@ -162,7 +162,7 @@ def handle_ip_option(env)
             if @interface_network[:created]
               # Just check for mismatch error here - if name and ip from
               # config match together.
-              if @options[:network_name] != @interface_network[:name]
+              if @options[:network_name] != @interface_network[:name] and @qemu_use_agent = false
```

Comment:
@abbbi I missed that this is actually setting @qemu_use_agent to false rather than comparing, it should use `==`

Reply:
whoops :)